plover/dictionary/rtfcre_dict.py

```diff
@@ -307,8 +308,9 @@ def load_dictionary(filename):
           "{\\stylesheet{\\s0 Normal;}}\r\n")
 
 def format_translation(t):
-    t = ' '.join([x.strip() for x in META_RE.findall(t) if x.strip()])
```

Comment:
Messing with the whitespace seemed odd. If whitespace doesn't matter, then why bother modifying it?